templates/pasta.html

```diff
@@ -48,6 +66,23 @@
   user-select: none;
 }
 
+.code-container {
+  position: relative;
+}
+
+.hidden {
+  display: none;
+}
+
+.copy-button {
+  position: absolute;
+  background: transparent;
+  top: 0;
+  right: 0;
+  padding: 8px;
+  margin: 5px;
```

Comment:
Adjusted placing for the new shape, plus change cursor type to show it's interactive:

```
padding: 3px;
margin: 3px;
cursor: pointer;
```